project/Dependencies.scala

```diff
@@ -21,7 +21,7 @@ object Dependencies {
   val cats           = "org.typelevel"               %% "cats-core"                   % Version.cats
   val scalactic      = "org.scalactic"               %% "scalactic"                   % Version.scalactic
   val scalatest      = "org.scalatest"               %%  "scalatest"                  % Version.scalatest % "test"
-  val jaiCore        = "javax.media"                 % "jai_core"                     % "1.1.3"
+  val jaiCore        = "javax.media"                 % "jai_core"                     % "1.1.3" from "https://repo.osgeo.org/repository/release/javax/media/jai_core/1.1.3/jai_core-1.1.3.jar"
```

Comment:
This is the change to make sbt happy with the jai_core dep. In the new GT releases it would not be neccesary to add jai_core like this.

However, this fix does not guarantee that all builds would sucesfully download it. To guarantee it we need to exclude jai_core from all deps to avoid accidential bad resolutions. 

I experimented with it and cam to a conclusion that we can exclude all jai_core deps from all the 3d party deps and in this case we guarantee the resolution odering. [That is the fix for GeoTrellis](https://github.com/locationtech/geotrellis/blob/master/project/Settings.scala#L278).

Reply:
Gotcha, so I still need to add the exclude in your link to the build.sbt for vectorpipe.

Reply:
@CloudNiner well we can live without it for now I guess